common/kvstore/src/main/java/org/apache/spark/util/kvstore/InMemoryStore.java

```diff
@@ -164,7 +164,7 @@ public void clear() {
   }
 
   /**
-   * An alias class for the type "ConcurrentHashMap<Comparable<Object>, Boolean>", which is used
+   * An alias class for the type "{@literal ConcurrentHashMap<Comparable<Object>, Boolean>}", which is used
```

Comment:
Looks like this line is too long.

Reply:
You're right, failed on checkstyle. Thank you.
Fixed.

Reply:
@gemelen seems forgot to push the change?